### DISCUSSION TITLE
How to create a project

### DISCUSSION THREAD

**Discussion by Doid00Yeah** [HaxeFlixel/flixel#2326] [Q&A] [answered]
I am using the command flixel tpl -n "Example", but I cannot create the project.
When I check the folder there is nothing and vscode doesn't open.
And that appears:
![prompt](https://user-images.githubusercontent.com/86123302/122595353-b6e5df00-d03e-11eb-8546-bda68c6edb4f.png)
Sorry im new here and i dont know how to put a image.

**Comment by Gama11** [comment, marked as answer]
Don't try to create projects in the System32 folder.

**Reply by Doid00Yeah** [reply]
Okay, where would be the best place to create a project?

**Reply by Gama11** [reply]
Almost any other places should work. Somewhere in your home directory is probably most common.

**Reply by Doid00Yeah** [reply]
Ok, thanks.
